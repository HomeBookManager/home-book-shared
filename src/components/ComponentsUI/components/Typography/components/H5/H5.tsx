import { FC } from 'react';

// components
import Typography from '../../Typography';

// types
import { TOmittedTypographyProps } from '../../types';
import { TypographyFontType } from '../../enums';

const H5: FC<TOmittedTypographyProps> = ({ children, ...restProps }) => (
  <Typography fontType={TypographyFontType.h5} {...restProps}>
    {children}
  </Typography>
);

export default H5;
