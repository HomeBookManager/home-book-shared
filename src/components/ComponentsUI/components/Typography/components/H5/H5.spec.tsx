import { render } from '@testing-library/react';

// components
import H5 from './H5';

describe('H5 snapshots', () => {
  it('should render H5', () => {
    //before
    const { asFragment } = render(<H5>H5</H5>);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
