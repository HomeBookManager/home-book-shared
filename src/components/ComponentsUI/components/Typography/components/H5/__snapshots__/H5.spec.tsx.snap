// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H5 snapshots should render H5 1`] = `
<DocumentFragment>
  <h5
    class="Typography Typography--regular Typography--normal Typography--h5 Typography--without-margin"
    data-test-text=""
    style="color: rgb(30, 40, 57); text-align: inherit;"
  >
    H5
  </h5>
</DocumentFragment>
`;
